.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*     @googleapis/actools @googleapis/oh-eng @googleapis/yoshi-python
+*     @googleapis/actools @googleapis/oh-eng @googleapis/yoshi-python @googleapis/cloud-client-library-release-automation-team
```

Comment:
Commenting since there is no presubmit for this: looks like the team is private or DNE - it's underlined as invalid in code review here

Reply:
Needed repo write access. Now passing validity check.